spring-batch-docs/asciidoc/domain.adoc

```diff
@@ -665,7 +801,7 @@ not be written out. More details about the `ItemProcessor` interface can be
 found in <<readersAndWriters.adoc#readersAndWriters,Readers And Writers>>.
 
 
-=== Batch Namespace
+=== Batch Namespace When Using XML Configuration
```

Comment:
I'm wondering if we should just hide this when the java config toggle is flipped...